File: reminderbot.py
import discord
import datetime
from apscheduler.schedulers.background import BackgroundScheduler

from discord.ext.commands import Bot
my_bot = Bot(command_prefix="!")
@my_bot.event
async def on_read():
    print("Client logged in")
@my_bot.command()
async def hello(*args):
    return await my_bot.say("Hello, world!")

@my_bot.command()
async def setgb():
	print("notifying")
	date = datetime.datetime.now()
	scheduler = BackgroundScheduler()
	#scheduler.add_job(my_bot.say("GB!"),'interval',seconds = 15)
	notify()
	scheduler.start()
	return
		

@my_bot.command()
async def notify():
	print("notifying")
	return my_bot.say("GB!")


my_bot.run("MjkwNjU5NDAwOTE2NjY0MzIx.C6eKfg.zHWCzpQWb6G0YM4NCwQrii9Ptm8")

File: reminderbot.js
var DiscordClient = require('discord.io');
var bot = new DiscordClient.Client({
    autorun: true,
    email: "", // email
    password: "", // password
    //OR
    token: "MjkwNjU5NDAwOTE2NjY0MzIx.C6eKfg.zHWCzpQWb6G0YM4NCwQrii9Ptm8"
});

bot.on('ready', function() {
    console.log(bot.username + " - (" + bot.id + ")");
});

bot.on('message', function(user, userID, channelID, message, rawEvent) {
    if (message === "!setGBReminders") { // command to trigger
    	var date = new Date();
    	var year = date.getFullYear();
    	var day = date.getDate();
    	var hour = date.getHours();
    	var min = date.getMinutes();
    	var sec = date.getSeconds();
    	var eta_ms = new Date(year,day,10,55,0) - Date.now();
    	var timeout = setTimeout(function(){
      		var interval = setInterval (function (){
		        bot.sendMessage({
		          to: channelID,
		          message: "GB!" // message to send
		        });
      		}, 86400000); // time between each interval in milliseconds
      	},eta_ms);

      	var eta_ms = new Date(year,day,15,55,0) - Date.now();
    	var timeout = setTimeout(function(){
      		var interval = setInterval (function (){
		        bot.sendMessage({
		          to: channelID,
		          message: "GB!" // message to send
		        });
      		}, 86400000); // time between each interval in milliseconds
      	},eta_ms);

      	var eta_ms = new Date(year,day,18,55,0) - Date.now();
    	var timeout = setTimeout(function(){
      		var interval = setInterval (function (){
		        bot.sendMessage({
		          to: channelID,
		          message: "GB!" // message to send
		        });
      		}, 86400000); // time between each interval in milliseconds
      	},eta_ms);

      	bot.sendMessage({
		          to: channelID,
		          message: "GB Reminders set!" // message to send
		        });
    }
    // New command starts here
    else if (message === "!help") {
      bot.sendMessage({
        to: channelID,
        message: "Insert Message here"
      });
    }
    // Ends here
});